src/playground.ts

```diff
@@ -12,10 +13,103 @@ if (configurations.email.enabled) {
 }
 
 import { sync } from './database';
+import schedulerHelper, { HttpMethod } from './utilities/scheduler-helper';
+
+const sleep = (millis: number): Promise<void> => new Promise(resolve => setTimeout(resolve, millis));
```

Comment:
_immediately suspicious_

Reply:
Haha it's just for the test, but yes you are correct!